vignettes/newfeatures.Rmd

```diff
@@ -0,0 +1,207 @@
+---
```

Comment:
This vignette will be really useful for the next release and I think its correct display is definitly on your blog and not in the package. 
However, what we should do is to add 3 new entries in the current vignette (vignettes/cartography.Rmd).

Reply:
Thank you! May I ask then to include myself as author of the vignette? I would kindly ask also to to be considered as a "ctb" (Contributor)  of the package, just me let know your thoughs on that.

Once that you let me know what to do with `tilesLayer`/`pngLayer` I could make a proposal of the vignette.

Reply:
Yes of course! For the vignette and for the package...
I'll make the change to imageLayer after merging the PR

Reply:
Nice 👍

Reply:
Nice 👍

Reply:
I've finally decided to keep both functions (one is only an alias of the other). You can keep the blog post as is.

Reply:
Fantastic, thank you